### PR TITLE
Product not found toast in search should be removed

### PR DESCRIPTION
**Pull request by shivanshsoni**
fixed issue #890

### CODE REVIEW

File: app/src/test/java/openfoodfacts/github/scrachx/openfood/network/OpenFoodAPIServiceTest.java [hunk 1]
```diff
@@ -198,7 +198,7 @@ public void getProduct_notFound() throws Exception {
         assertTrue(response.isSuccessful());
 
         assertEquals(0, response.body().getStatus());
-        assertEquals("product not found", response.body().getStatusVerbose());
+        //assertEquals("product not found", response.body().getStatusVerbose());
```
Comment (PrajwalM2212) [inline app/src/test/java/openfoodfacts/github/scrachx/openfood/network/OpenFoodAPIServiceTest.java]: This is a test class and does not affect the behaviour of the app.

**Comment by shivanshsoni** [comment]
Ya Sir, I know but somehow i commited wrong changes.... I have fixed them now

**Comment by PrajwalM2212** [comment]
@shivanshsoni  No problem . Take your time .

**Comment by policorn** [comment]
Thanks @shivanshsoni! PR merged

**Pull request merged**